app/src/Service/saveService.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\src\Service;
+
+use Audubon\Entities\Bird;
+use Audubon\Entities\Capture;
+use Audubon\Entities\Member;
+use LaravelDoctrine\ORM\Facades\EntityManager;
+
+class saveService
+{
+    private Bird $birdObj;
+    private Capture $captureObj;
+    private Member $memberObj;
+
+    public function putDataIntoClasses($data): void
+    {
+        $this->birdObj = $this->getBirdData($data);
```

Comment:
Unless you have a few bird related variables, I think obj is unnecessary.  There may be some cases where you access it many times on various functions where it makes sense to make it a global variable.  In this case, you'd only have to pass it to one function, so I'd just make it a local variable.  $bird, $member, $capture.  That is the syntax for creating a local variable.